controlnet-1.5-openpose/predict.py

```diff
@@ -120,9 +125,20 @@ def predict(
 
         if os.path.exists("control.png"):
             os.unlink("control.png")
-        shutil.copy(control_image, "control.png")
-        image = load_image("control.png")
-        image = self.openpose(image)
+        
+        if control_image is not None:
+            print("Using ControlNet pipeline")
+            shutil.copy(control_image, "control.png")
+            image = load_image("control.png")
+            image = self.openpose(image)
+            pipe = self.controlnetpipe
+            extra_kwargs = {
+                "image": image,
+            }
+        else:
+            print("Using txt2img pipeline")
+            pipe = self.txt2img_pipe
+            extra_kwargs = {}
```

Comment:
Here we check is the `control_image` is set, if it is set we use ControlNet pipeline, otherwise we fall back to txt2img.

The `extra_kwargs` variable comes frome https://github.com/anotherjesse/dream-templates/blob/main/vanilla/predict.py#L126